### DISCUSSION TITLE
Create Bounty program

### DISCUSSION THREAD

**Discussion by inson1** [mullvad/mullvadvpn-app#8957] [Feature requests & Ideas]
I think apps with bounty programs are safer. It doesnt cost that much. It would incentivize more developers to try find security risks and vulnerabilities.
Plus its great advertisement almost for free. 

https://www.reddit.com/r/VPN/comments/m736zt/vpn_comparison_table/
![image](https://github.com/mullvad/mullvadvpn-app/assets/75314629/470d7802-6ec4-4847-af5c-26d6c4972381)

**Comment by faern** [comment]
It's not a cost issue, but rather a time issue. Someone has to respond to incoming reports and verify them. That's not a problem for valid true issues and vulnerabilities. But when you bring money into the equation you are unavoidably going to get semi-flooded with not-very-important-issues. And someone has to judge whether or not a specific issue is bounty-worthy or not. There is unavoidably going to be disputes where the reporter claim they should get cash but we really don't think the finding is a real threat in any way.
